index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Ayush's Portfolio</title>
+</head>
+<body>
+    <div class="container">
+        <div class="welcome-container">
+            <!--Header Begins-->
+            <div class="toolbar-container">
+                <header class="toolbar">
+                    <div class="logo-container">
+                        <a href="#" class="logo">Ayush Jha</a>
+                    </div>
+                    <div class="menu-container">
+                        <a href="#" class="menu"><img src="./assets/images/Union.svg" alt="" class="menu-image"></a>
+                    </div>
```

Comment:
Kindly change this `div` to a `nav` tag 😃

- _The `nav` tag is for semantics reasons. Later in the project, you will add a navigation menu that opens when you click the hamburger._